Assets/HoloToolkit/Input/Scripts/OnGazeEnterEvent.cs

```diff
@@ -1,21 +0,0 @@
-﻿using UnityEngine;
```

Comment:
Just wondering why did you delete the *Event files ?
Interactable doesn't give you the option to setup Unity Event straight from the inspector (unless I'm missing something) and that was all the point of those files..